src/Functions/FunctionBinaryArithmetic.h

```diff
@@ -1326,6 +1421,21 @@ class FunctionBinaryArithmetic : public IFunction
             return getReturnTypeImplStatic(new_arguments, context);
         }
 
+        if (isArray(arguments[0]) && isArray(arguments[1]))
```

Comment:
Shouldn't we constrain this only for plus and minus functions?

Reply:
This shouldn't be allowed:
```
ip-10-2-3-206.eu-west-1.compute.internal :) SELECT [12,13] % [5,6]

SELECT [12, 13] % [5, 6]

Query id: 62e9c613-b2d5-44e5-9f29-0abd51cd17df

┌─modulo([12, 13], [5, 6])─┐
│ [2,1]                    │
└──────────────────────────┘

1 row in set. Elapsed: 0.007 sec.
```

Reply:
yes

Reply:
Yes, only for plus and minus